llm_toolkit/models.py

```diff
@@ -56,6 +56,8 @@ class LLM:
   # TODO(mihaimaruseac): Should this be MAX_TOKENS or a different global?
   context_window: int = 2000  # Default token size.
 
+  MAX_INPUT_TOKEN: int = 0
```

Comment:
should this be infinity by default? to avoid surprising behaviour otherwise. It seems to me that the default should be to have no limit, rather than limit to 0?

Reply:
Thanks, using `sys.maxsize` now.